ui/plugin-system/src/stories/shared-utils/decorators/WithPluginRegistry.tsx

```diff
@@ -19,12 +19,8 @@ import {
   dynamicImportPluginLoader,
 } from '@perses-dev/plugin-system';
 
-// NOTE: the aliases we use for components break these top level imports, so we
-// import relatively.
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const prometheusResource = require('../../../../../prometheus-plugin/plugin.json');
-// eslint-disable-next-line @typescript-eslint/no-var-requires
-const panelsResource = require('../../../../../panels-plugin/plugin.json');
+import prometheusResource from '@perses-dev/prometheus-plugin/plugin.json';
+import panelsResource from '@perses-dev/panels-plugin/plugin.json';
```

Comment:
I initially said I'd do this in a separate PR because I thought I'd need to set up more aliases and refactor `storybook/config/main.js` a bunch, but it turns out the tweaks I made to the alias setup in https://github.com/perses/perses/pull/1121 fixed this for me. I just needed to switch the imports back.